base/doc/ltnews37.tex

```diff
@@ -149,6 +149,29 @@ \section{New or improved commands}
 %
 \githubissue{xxx}
 
+\subsection{Extension of the \cs{label} command}
+
+Up to now the \cs{label} command stored in standard \LaTeX{} into the
+\cs{newlabel} command in the \texttt{.aux} file two data: 
+\cs{@currentlabel}, which normally contains the state of the current 
+counter, and \cs{thepage} for the current page number.  \pkg{hyperref} and 
+\pkg{nameref} patch the \cs{label} command to store five data: 
+Additionally they save \cs{@currentlabelname}, which normally contains the 
+current title text and can be retrieved with \cs{nameref} and 
+\cs{@currentHref} which is the name of the destination needed to create an 
+active link. The fifth data container is currently only used if external 
+references are loaded with the \pkg{xr-hyper} package. 
+
+Starting with this release the number of arguments have be unified. 
+\cs{label} now writes a
+\cs{newlabel} command containing five data: \cs{@currentlabel},
```

Comment:
with five arguments

Reply:
No! They are not (and in ltxref never become) arguments of anything --

Suggestion: ... command that stores, in its second argument, the following five items of data:

Reply:
container ==>  (data) item